Sources/MessagingPush/RichPush/RichPushHttpClient.swift

```diff
@@ -160,8 +160,10 @@ extension RichPushHttpClient {
 
     static func getBasicSession() -> URLSession {
         let configuration = URLSessionConfiguration.ephemeral
-        configuration.httpMaximumConnectionsPerHost = 2
         configuration.allowsCellularAccess = true
+        configuration.timeoutIntervalForResource = 30
+        configuration.timeoutIntervalForRequest = 60
+        configuration.httpAdditionalHeaders = [:]
```

Comment:
Why were these lines added?

Reply:
they are in `main` so I think they were adding values in there.